tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -0,0 +1,186 @@
+"""
+.. _tut-fnirs-processing:
+
+Preprocessing functional near-infrared spectroscopy (fNIRS) data
+================================================================
+
+This tutorial covers how to convert functional near-infrared spectroscopy
+(fNIRS) data from raw measurements to relative oxyhaemoglobin (HbO) and
+deoxyhaemoglobin (HbR) concentration.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+Here we will work with the :ref:`fNIRS motor data <fnirs-motor-dataset>`.
+"""
+# sphinx_gallery_thumbnail_number = 3
+
+import os
+import numpy as np
+from itertools import compress
+import matplotlib.pyplot as plt
+
+import mne
+
+
+fnirs_data_folder = mne.datasets.fnirs_motor.data_path()
+fnirs_raw_dir = os.path.join(fnirs_data_folder, 'Participant-1')
+raw_intensity = mne.io.read_raw_nirx(fnirs_raw_dir, verbose=True).load_data()
+
+
+###############################################################################
+# Selecting channels appropriate for detecting neural responses
+# -------------------------------------------------------------
+#
+# First we remove channels that are too close together to detect a neural
+# response. To achieve this we pick all the channels that are not considered
+# to be short (less than 1 cm distance between optodes).
+
+short_channels = mne.preprocessing._beer_lambert_law._short_channels(
```

Comment:
calling a private function in an example should be fixed quickly  in  a next PR

Reply:
Don't worry, [that's the plan](https://github.com/mne-tools/mne-python/pull/7018#issuecomment-550823426)